style.css

```diff
@@ -0,0 +1,959 @@
+}
+
+.logo a {
+  height: 40px;
+  width: 20px;
+  text-decoration: none;
+  font-size: 20px;
+  font-weight: 700;
+  font-family: "Inter", sans-serif;
+  color: #eee;
+  cursor: pointer;
+}
+
+.logo a span {
+  color: #0ff;
+}
+
+.nav-links {
+  display: flex;
+  list-style: none;
+}
+
+.nav-links li {
+  padding: 0 20px;
+}
+
+.nav-links a {
+  width: 30px;
+  height: 20px;
+  font-size: 25px;
+  font-weight: 500;
+  color: #eee;
+  align-items: center;
+  justify-content: center;
+  transition: transform 0.4s;
+}
+
+.nav-links li a {
+  transition: all 0.3s ease 0s;
+  width: 63px;
+  height: 20px;
+  font-family: "Inter", sans-serif;
+  font-style: normal;
+  font-weight: 600;
+  font-size: 15px;
+  line-height: 20px;
+  color: #eee;
+  text-decoration: none;
+}
+
+footer .social a {
+  font-size: 25px;
+  padding: 20px;
+  color: #505f79;
+}
+
+.nav-links a:hover {
+  color: #0ff;
+  transform: scale(1.2);
+}
+
+nav .vector {
+  width: 18px;
+  height: 10px;
+  background: #3c3a39;
+  top: 30px;
+  right: 5%;
+  z-index: 5;
+  position: absolute;
+  display: none;
+}
+
+nav .vector span {
+  display: block;
+  width: 100%;
+  background: #fff;
+  height: 2px;
+}
+
+nav .vector span:nth-child(2) {
+  margin: 3px 0;
+}
+
+footer .social {
+  width: 100%;
+  height: 25px;
+  display: inline-flex;
+  align-items: center;
+  justify-content: center;
+  font-size: 25px;
+  color: rgb(145, 143, 143);
+  transition: all 0.3s ease 0s;
+}
+
+.portfolio .social a {
+  font-size: 20px;
+  padding-top: 10px;
+  color: #eee;
+}
+
+footer .social a:hover {
+  color: #f55800;
+}
+
+.nav-links li a:hover {
+  color: #0ff;
+}
+
+.portfolio {
+  display: flex;
+  flex-wrap: nowrap;
+  align-items: center;
+  margin-top: 100px;
+  margin-left: 55px;
+}
+
+.portfolio > div {
+  display: flex;
+  flex-direction: column;
+  margin: 10px;
+}
+
+h1 {
+  width: 280px;
+  height: 150px;
+  font-family: "Crete Round", serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 45px;
+  line-height: 56px;
+  padding-top: 30px;
+  color: #f55800;
+  flex: none;
+  order: 0;
+  flex-grow: 0;
+}
+
+.portfolio h2 {
+  width: 300px;
+  height: 35px;
+  font-family: "Inter", sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 25px;
+  color: #fff;
+  flex: none;
+  order: 1;
+  flex-grow: 0;
+}
+
+.portfolio p {
+  width: 550px;
+  height: 100px;
+  font-family: "Inter", sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 17px;
+  line-height: 30px;
+  color: #f4f5f7;
+  flex: none;
+  order: 2;
+  flex-grow: 0;
+}
+
+.portfolio .social {
+  display: flex;
+  flex-direction: column;
+  transition: transform 0.4s;
+}
+
+.portfolio .social a:hover {
+  color: #0ff;
+  transform: scale(1.2);
+}
+
+.group {
+  background-image: url("./images/Illustration-header.png");
+  background-repeat: no-repeat;
+  background-size: contain;
+  width: 486px;
+  height: 500px;
+  position: absolute;
+  left: 66.5%;
+  top: 62px;
+}
+
```

Comment:
Great job adding the background image 👍, kindly update the styling as it is currently not responsive and also, the background image is overflowing 🙂
<img width="764" alt="image" src="https://user-images.githubusercontent.com/73854999/208181740-0a113ac3-4926-472b-9a3c-b11a9d462091.png">